dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/ConnectionPoolImpl.java

```diff
@@ -197,10 +197,10 @@ public boolean removeHost(Host host) {
 			cpHealthTracker.removeHost(host);
 			cpMonitor.hostRemoved(host);
 			hostPool.shutdown();
-            Logger.info(String.format("Remove host: Successfully removed host %s from connection pool", host.getHostName()));
+            Logger.info(String.format("Remove host: Successfully removed host %s from connection pool", host.getHostAddress()));
```

Comment:
Either the hostname or the private IP address will be printed. Based on what has been used the correct one will be printed. I am not sure I am following your question.